Gruntfile.js

```diff
@@ -11,13 +11,6 @@ module.exports = function(grunt) {
       }
     },
 
-    jshint: {
```

Comment:
This will fix the issue:

```
jshint: {
      src: ['./lib'],
      options: {
        reporterOutput: '',
        jshintrc: './jshintrc.js'
      }
    }
```